### PR TITLE
deployment: release v0.8.0

### PR DESCRIPTION
**Pull request by desimone**
## **Checklist**:

**Before tagging**

- [x] Bump version
- [x] Update docs to have a new version in config.js and in netlify
- [x] Review and update examples 
- [x] Review and update dockerfiles 
- [x] Review and update docs
- [x] Review and update changelog
- [x] Review and update upgrading guide

**After tagging**
- [ ] Review and update helm (merge after tag) 
- [ ] Update github release docs
- [ ] Set `latest` branch to track the new release
- [ ] Publish tagged version of documentation
- [ ] Review and update helm-charts (merge after tag) 
- [ ] Change docker examples back to `master` on master.

### CODE REVIEW

File: docs/_posts/2020-05-11-release-0-8.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+- [**Automatic Certificate Management**] — Pomerium can now be configured to automatically retrieve and renew certificates, adding HTTPS to all Pomerium managed routes. In addition, Pomerium will do [OCSP stapling](https://en.wikipedia.org/wiki/OCSP_stapling) for automatic and custom certificates alike.
+- [**Advanced Route Matching**] — Operators can now write access policy that supports route matching based on [regex], [prefix], and [path] settings. Pomerium now has the flexibility to support multiple and layered authorization policies across a single managed route.
+- And finally, this release adds [**Github**](https://github.com/) as a supported identity provider.
+
+Pomerium had 95 commits from 8 authors across 5 organizations in this release. This release also includes additional new features, general improvements, and bug fixes, a complete list of which can be found in the [changelog].
+
+As always, we recommend upgrading and testing this release in an isolated environment. If you experience any issues, please report them on the Pomerium GitHub [issue tracker].
+
+<SimpleNewsletter/>
```
Comment (travisgroth) [inline docs/_posts/2020-05-11-release-0-8.md:23]: Is this tag meant to be here?

File: docs/docs/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+
+To see a complete list of changes [see the diff](https://github.com/pomerium/pomerium/compare/v0.7.0...v0.8.0).
+
+### New
+
+- cryptutil: add automatic certificate management @desimone [GH-644]
+- implement path-based route matching @calebdoxsey [GH-615]
+- internal/identity: implement github provider support @Lumexralph [GH-582]
+
+### Changes
```
Comment (travisgroth) [inline docs/docs/CHANGELOG.md:13]: Forward auth support for the claim headers is in this release as well.  Not sure if that goes under changes or new, though.

File: docs/_posts/2020-05-11-release-0-8.md (hunk 1 above)
Comment (desimone) [reply docs/_posts/2020-05-11-release-0-8.md:23]: Yes, it adds the newsletter at the bottom :)

Comment (travisgroth) [reply docs/_posts/2020-05-11-release-0-8.md:23]: 👍

**Pull request merged**